### PR TITLE
docUpdate

### PR DESCRIPTION
**Pull request by StefanHolst**
Updated documentation to include more details about the naming of a package.

### CODE REVIEW

File: doc/Developer Guide/Plugin Packaging and Versioning/Readme.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ The root element of the configuration file is `Package` and it supports the foll
...
+| **Name** | We advise to make the names human readable, i.e. not Camel Case, no underscores, etc. No need to add "plugin" in the package name, as OpenTAP packages typically contain OpenTAP plugins. The package name determines where the package definition will be installed, relative to the Packages directory in the OpenTAP installation. E.g. the package definition of a package named `MyDutDriver` will be installed to `./Packages/MyDutDriver/package.xml`. Forward slashes are strongly discouraged because they will never be supported by the package repository. The following characters are not allowed: `"`, `<`, `>`, `\|`, `\0`, `\u0001`, `\u0002`, `\u0003`, `\u0004`, `\u0005`, `\u0006`, `\a`, `\b`, `\t`, `\n`, `\v`, `\f`, `\r`, `\u000e`, `\u000f`, `\u0010`, `\u0011`, `\u0012`, `\u0013`, `\u0014`, `\u0015`, `\u0016`, `\u0017`, `\u0018`, `\u0019`, `\u001a`, `\u001b`, `\u001c`, `\u001d`, `\u001e`, `\u001f`, `:`, `*`, `?`, `\`. |
```
Comment (alnlarsen) [inline doc/Developer Guide/Plugin Packaging and Versioning/Readme.md:49]: I agree with the sentiment, but it's a bit hypocritical of us that literally the next sentence uses `MyDutDriver` as the example of a package name

File: doc/Developer Guide/Package Publishing/Readme.md [hunk 2]
```diff
@@ -5,12 +5,16 @@ Package Publishing
 This is a list of point to check prior to published your package on [packages.opentap.io](https://packages.opentap.io).
 
 ### Update the Package Definition (`package.xml`) file:
+* Provide a meaningful [`Name`](https://doc.opentap.io/Developer%20Guide/Plugin%20Packaging%20and%20Versioning/#attributes-in-the-configuration-file). We advise to make the names human readable, i.e. not Camel Case, no underscores, etc. No need to add "plugin" in the package name, as OpenTAP packages typically contain OpenTAP plugins. The name cannot contain '/' or be changed later, if you want to change the name you must publish the package again.
```
Comment (alnlarsen) [inline doc/Developer Guide/Package Publishing/Readme.md:8]: Here we say that forward slashes are not supported, but in the `Plugin Packaging And Versioning` section, we say that they are "strongly discouraged". Let's be consistent here

**Review by alnlarsen — CHANGES_REQUESTED** [review]
Looks good. A few minor comments

File: doc/Developer Guide/Package Publishing/Readme.md (hunk 2 above)
Comment (StefanHolst) [reply doc/Developer Guide/Package Publishing/Readme.md:8]: This is regarding publishing, in this case a / is not supported. I would say it's the correct description

**Review by alnlarsen — APPROVED** [review]
Looks good

**Review by AsgerIversen — APPROVED** [review]

**Pull request merged**